MLS.Agent/CommandLine/CommandLineParser.cs

```diff
@@ -181,6 +181,11 @@ RootCommand StartInTryMode()
                                       "Enable verbose logging to the console",
                                       new Argument<bool>()));
 
+                command.AddOption(new Option(
+                                        "--port",
+                                        "Specify the port to launch the dotnet try process",
+                                        new Argument<int>()));
```

Comment:
A default value can be specified here. This is more helpful because it can be displayed in help and in the parse diagram.